components/Card.vue

```diff
@@ -52,6 +52,7 @@
         <div
           v-if="overlay"
           class="card-overlay d-flex justify-center align-center"
+          nuxt
         >
           <v-btn fab color="primary" :to="`/item/${item.Id}/play`">
```

Comment:
```suggestion
        >
          <v-btn fab color="primary" nuxt :to="`/item/${item.Id}/play`">
```

`div` doesn't have a nuxt attribute, but v-btn does: https://vuetifyjs.com/en/api/v-btn/#props :)

Reply:
@mrtimscampi Oops that's what happens when you cherry pick and doesn't pay much attention to the conflicts 😋.

I'll also take some time to check each v-btn and add the nuxt property to the links, as probably there are more than this one.